components/StreamField/blocks/NewsBlock/blocks.tsx

```diff
@@ -0,0 +1,26 @@
+import type { BlogPostValue } from './types';
+import styles from './NewsBlock.module.scss';
+
+const DateOptions = { year: 'numeric', month: 'long', day: 'numeric' } as const;
+
+const formatDate = (date: string) => {
+    const dateObject = new Date(date);
+    return new Intl.DateTimeFormat(undefined, DateOptions).format(dateObject);
```

Comment:
> Instead of having to declare DateOptions, we can just put the object here since we probably won't need it anywhere else.

Good!

> Also instead of undefined, will using [] work?

I was planning to use "en-GB" to keep the format consistent. I've done it now.